frontend/src/components/service/Details.tsx

```diff
@@ -100,8 +100,10 @@ export default function ServiceDetails() {
                       },
                       {
                         label: t('translation|Addresses'),
-                        getter: endpoint => endpoint.getAddressesText(),
-                        cellProps: { style: { width: '40%', maxWidth: '40%' } },
+                        getter: endpoint => {
+                          return endpoint.getAddressesText();
+                        },
```

Comment:
This change is not needed. As the code deleted does the same.

Reply:
Thank you so much for the feedback , I am really sorry for overlooking the guidelines.

Reply:
No worries. We definitely encourage folks to submit PRs even if they need a bit adjustment to get them merged.
I saw you closed this PR. Of course you can reopen a new one when you have the new changes, but usually you just have to amend your commits and force push. If you need help with this, join our [Slack channel](https://kubernetes.slack.com/messages/headlamp) and I can help.

This is an important fix that I would like to get in the next version. Are you able to send us a new commit after my review?

Reply:
Ah, I see that there's another PR that's related to this and we should use that one. Sorry for the confusion @Src0p . I hope to see more contributions from you!